apps/emqx_connector/test/emqx_connector_api_SUITE.erl

```diff
@@ -84,7 +84,7 @@ suite() ->
 
 init_per_suite(Config) ->
     ok = emqx_config:put([emqx_dashboard], #{
-        default_username => <<"admin">>,
+        default_username => <<"connector_admin">>,
```

Comment:
Is this part now needed? (Since the initialization takes place in set_special_configs fun)

Reply:
Removed